.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -62,35 +86,23 @@ You can use `fastmod` to interactively find and replace versions.
 
 For example, you can do something like:
 ```
-fastmod --extensions rs,toml,md --fixed-strings '1.0.0-alpha.12' '1.0.0-alpha.13'
-fastmod --extensions rs,toml,md --fixed-strings '0.2.9' '0.2.10' tower-batch
-fastmod --extensions rs,toml,md --fixed-strings '0.2.8' '0.2.9' tower-fallback
+fastmod --extensions rs,toml,md --fixed-strings '1.0.0-beta.11' '1.0.0-beta.12'
+fastmod --extensions rs,toml,md --fixed-strings '0.2.26' '0.2.27' tower-batch tower-fallback
 ```
 
-### Reviewing Version Bumps
-
-Check for missed changes by going to:
-`https://github.com/ZcashFoundation/zebra/tree/<commit-hash>/`
-Where `<commit-hash>` is the hash of the last commit in the version bump PR.
-
-If any Zebra or Tower crates have commit messages that are **not** a version bump, we have missed an update.
-
-Also check for crates that depend on crates that have changed. They should get a version bump as well.
+Ignore any `fastmod` updates to previous release notes in `CHANGELOG.md`.
```

Comment:
It is not clear what this sentence means to me.

Reply:
Fixed in 181cd649e6e83893fa6f9fb0be8fd94ecefd3056